hackathon_site/dashboard/frontend/src/components/dashboard/EditTeam/EditTeam.tsx

```diff
@@ -36,41 +45,72 @@ const TeamInfoBlock = ({ teamCode, canChangeTeam, teamSize }: TeamModalProps) =>
     );
 };
 
-const TeamChangeForm = ({ canChangeTeam }: TeamModalProps) => {
+const TeamChangeForm = ({ canChangeTeam, teamCode }: TeamModalProps) => {
     const [value, setValue] = React.useState("");
-
+    const dispatch = useDispatch();
+    const isJoinTeamLoading = useSelector(isJoinLoadingSelector);
     const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
         setValue(event.target.value);
     };
+
+    const handleSubmitExternal = async (values: FormikValues) => {
+        console.log(values);
+        dispatch(joinTeam(values.teamCode));
```

Comment:
The redux stuff looks great! the only minor thing is that when a user joins a new team, the new team's orders are not displayed. So could you dispatch `getTeamOrders` from orderSlice.ts right after you dispatch the joinTeam thunk